functions/arrayLoop/1.0/index.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-await-in-loop */
+const loop = async ({ array }, steps) => {
```

Comment:
What if the value is null? You can handle this via: `({ array = [] }, steps) => {`